filament/src/materials/tonemappingWithSubpass.mat

```diff
@@ -0,0 +1,81 @@
+material {
+    name : tonemap,
+    parameters : [
+        {
+            type : int,
+            name : dithering
+        },
+        {
+            type : int,
+            name : fxaa
+        },
+        {
+            type : float4,
+            name : bloom
+        }
+    ],
+    variables : [
+        vertex
+    ],
+    depthWrite : false,
+    depthCulling : false,
+    domain: postprocess,
+    framebufferFetch: true
+}
+
+vertex {
+    void postProcessVertex(inout PostProcessVertexInputs postProcess) {
+        postProcess.vertex.xy = postProcess.normalizedUV;
+    }
+}
+
+fragment {
+
+    // TODO: this should be specified as a parameter
+    layout (input_attachment_index = 0, set = 0, binding = 0) uniform mediump subpassInput colorBuffer;
```

Comment:
We should add new material properties to define those. Something like:
```
inputs: [
    { name: colorBuffer, type: subpassInput }
],
// etc.
```

Reply:
yes, but that will tie into MRTs which we also need something like that for.
e.g. I'm not sure it should be "inputs" because a subpass can be an output too.
i don't understand subpasses well enough at this point.